spec/index.bs

```diff
@@ -7239,6 +7257,22 @@ The following enums are supported if and only if the {{GPUFeatureName/"timestamp
         * {{GPUQueryType/"timestamp"}}
 </dl>
 
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>minmax-sampling</dfn> ## {#minmax-sampling}
```

Comment:
```suggestion
## <dfn enum-value for=GPUFeatureName>minmax-sampling</dfn> ## {#minmax-sampling}
```

Reply:
We should change this across all features, consistently (as a follow-up)